tools/unicode_case_conversion.py

```diff
@@ -395,7 +396,7 @@ def extract_character_pair_ranges(letter_case, reverse_letter_case):
             in_range = False
 
     # Remove all founded case mapping from the conversion tables after the scanning method
-    for idx in range(len(start_points)):
+    for idx, _ in enumerate(start_points):
```

Comment:
Yes, with the following msg: 
"Consider using enumerate instead of iterating with range and len (consider-using-enumerate)"